src/doc/layer_io.cpp

```diff
@@ -1,10 +1,11 @@
 // Aseprite Document Library
-// Copyright (c) 2019 Igara Studio S.A.
+// Copyright (c) 2019-2025 Igara Studio S.A.
 // Copyright (c) 2001-2018 David Capello
 //
 // This file is released under the terms of the MIT license.
 // Read LICENSE.txt for more information.
 
+#include "doc/uuid_io.h"
```

Comment:
Please move this `#include` below with the other ones.

`"config.h"` and `"doc/layer_io.h"` must be the first ones.